.github/workflows/generate-from-configuration.yaml

```diff
@@ -0,0 +1,113 @@
+name: Generate GAPIC libraries from configuration
+on:
+  schedule:
+  - cron: '0 2 * * *' # nightly at 2 am UTC
+  push:
+    branches:
+    - feat/hermetic-pr-creation
+  workflow_dispatch:
+
+
+jobs:
+  generate-from-configuration:
+    runs-on: ubuntu-22.04
+    env:
+      branch_name: generate-libraries
+      library_generation_image_tag: latest
+      repo_volumes: "-v repo-google-cloud-java:/workspace/google-cloud-java"
```

Comment:
Anything that references `google-cloud-java` should be extracted to a parameter, so that we can reuse this Github action in handwritten library repos.

Reply:
I'll leave this one as-is and we'll refactor it once we prove this is working on google-cloud-java.